baremetal-prep/baremetal-prep.sh

```diff
@@ -18,6 +18,11 @@ disable_selinux(){
 }
 
 setup_default_pool(){
+
+  #libvirt must be running
+  sudo service libvirtd restart
+  sleep 2
```

Comment:
I'd say it would be better to use `systemctl is-active libvirtd` and start it otherwise (vs service which is deprecated)

Reply:
thank you. I implemented as suggested and run a test with enabled/disabled:
https://github.com/ch-stark/baremetal-deploy/commit/3784b03c06629c23531275a384d3f6976c32c701
  #libvirt must be running
  if ( ! systemctl is-active libvirtd) then
    echo "starting libvirt"
    sudo systemctl start libvirtd
    sleep 2
  fi